projects/core/src/product/connectors/product/product.adapter.ts

```diff
@@ -0,0 +1,6 @@
+import { Observable } from 'rxjs';
+import { Product } from '../../../occ/occ-models/occ.models';
+
+export abstract class ProductAdapter {
+  abstract load(productCode: string): Observable<Product>;
```

Comment:
can we add a bit of documentation for this abstract method?

Reply:
👍